src/PlaneFitting.hpp

```diff
@@ -89,25 +89,25 @@ class PlaneFitting
 
     class Result
     {
-	Matrix3 A;
```

Comment:
This assumes that nobody will ever want to read A and b... Is it really a problem to leave it there ?